src/modules/pos/validator/components/ChangeCommission/Status/index.js

```diff
@@ -0,0 +1,19 @@
+/* istanbul ignore file */
+import { connect } from 'react-redux';
+import { selectActiveTokenAccount } from 'src/redux/selectors';
+import { transactionBroadcasted, resetTransactionResult } from 'src/redux/actions';
+import Status from './ChangeCommissionStatus';
+
+const mapStateToProps = state => ({
+  account: selectActiveTokenAccount(state),
+  transactions: state.transactions,
+});
+const mapDispatchToProps = {
+  transactionBroadcasted,
+  resetTransactionResult,
+};
```

Comment:
@soroushm These aren't used in the wrapped component. Since there are also selectors for the `mapStateToProps` maybe those can be used in the wrapped component